examples/elasticsearch-incremental/elasticsearch-incremental-blueprint.yaml

```diff
@@ -0,0 +1,555 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata: 
+  name: elasticsearch-incremental-blueprint 
+actions:
+  backup:
+    outputArtifacts:
+      s3Snap:
+        keyValue:
+          repoPath: '{{ .Phases.setupNodeKeyStore.Output.repoPath }}'
+          snapshotName: '{{ .Phases.setupNodeKeyStore.Output.snapshotName }}'
+          esClusterName: '{{ .Phases.setupNodeKeyStore.Output.esClusterName }}'
+          esClusterNamespace: '{{ .Object.metadata.namespace }}'
+    phases:   
+    - func: KubeTask
+      name: setupNodeKeyStore
+      objects:
+        elasticSecret:
+          kind: Secret
+          name: '{{ .Object.metadata.name }}-es-elastic-user'
+          namespace: '{{ .Object.metadata.namespace }}' 
+      args:        
+        image: ghcr.io/kanisterio/kanister-kubectl-1.18:0.81.0
+        command:
+        - bash
+        - -x
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          debug=0
+          function bp_dbg
+          {
+              # use 
+              # kubectl logs -n kasten-io -l component=kanister --tail=10000 -f | ggrep -o -P '(?<=Elasticbps3echobegin).*(?=Elasticbps3echoend)' 
+              # to grab the logs in kanisters
+              if [[ $debug -eq 1 ]]
+              then 
+                echo "Elasticbps3echobegin backup-action: $* Elasticbps3echoend"
+              fi
+          }
```

Comment:
I think this `bp_dbg` can be removed because we are not using this anywhere in this phase.

Reply:
bp_dbg has been removed